package.json

```diff
@@ -23,8 +23,10 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "eslint": "^5.15.1",
```

Comment:
Isn't just `gulp-eslint` enough? Why do we also need `eslint` itself?

Reply:
Because `gulp‑eslint` declares `eslint` as&nbsp;a&nbsp;`peerDependency` (or&nbsp;at&nbsp;least it&nbsp;did when I&nbsp;last checked)

Reply:
[ES-lint is a normal dependency](https://github.com/adametry/gulp-eslint/blob/master/package.json#L43) now.